### ISSUE TITLE
Should have an option to disable use of icons

### ISSUE THREAD

**Issue by BelfordZ** [Foxandxss/angular-toastr#170]
As the title suggests. Basically exactly like the close button boolean.

**Comment by Foxandxss** [comment]
I hear you but that is CSS and it is an entire class that does everything from color to icons. You would need to edit the CSS to delete it or create custom classes.

Check the [CSS](https://github.com/Foxandxss/angular-toastr/blob/master/src/toastr.less#L152-L166)

**Comment by BelfordZ** [comment]
Yes, but if you override those classes to not have any image, the spacing becomes off-looking.

**Comment by Foxandxss** [comment]
You can edit those classes too. It is all CSS.

To be honest, I don't know what I can do here to help you :/

**Comment by BelfordZ** [comment]
well yeah... that is what I did. I was suggesting this as something Im sure others have had to do as well.

**Comment by karlschwaier** [comment]
Yes, I had to remove the icons as well. Overriding the CSS was also what I did and it worked quite well. Anyway I agree that it would be nice to have an option to turn icons on and off. Maybe by moving the appropriate styles to a separate class like `toast-icon` and toggling it on `<div class="toast toast-info">`.

**Comment by Foxandxss** [comment]
Yeah, why not. I would like to rewrite the whole css to be able to remove replace. I can have this in mind.

**Comment by ruisilva450** [comment]
So? Any news on this?

**Comment by BelfordZ** [comment]
I think @Foxandxss doesn't work on this project anymore.

**Comment by Foxandxss** [comment]
I am sorry, I am super super busy with other stuff and I have to admit that this project is not on my top priority since I don't do Angular 1 anymore.

I should pick someone to help me and make this useful again.

**Comment by BelfordZ** [comment]
Neither do I, but I could help you out if you would like. Send me an email if you would like to chat about it.

**Comment by Foxandxss** [comment]
I appreciate that, but the last thing we need is two busy people.

I will try to assign this some time.
